test/record/record-handlerSpec.js

```diff
@@ -62,6 +62,51 @@ describe( 'record handler handles messages', function(){
 		expect( clientA.socket.lastSendMessage ).toBe( msg( 'R|R|existingRecord|3|{"firstname":"Wolfram"}+' ));
 	});
 
+	it( 'returns true for HAS if message exists', function(){
+		recordHandler.handle( clientA, {
+			raw: msg( 'R|H|existingRecord' ),
+			topic: 'RECORD',
```

Comment:
Good catch, I think I copied that from somewhere so will need to take a look where